.github/workflows/.env

```diff
@@ -0,0 +1,9 @@
+BITNAMI_TEAM='["bitnami-bot","jotamartos","alemorcuq","agomezmoron","carrodher","dgomezleon","fmulero","FraPazGal","javsalgar","gongomgra","joancafom","marcosbc","aoterolorenzo","CeliaGMqrz","Mauraza","mdhont","migruiz4","rafariossaa"]'
+SUPPORT_TEAM='["jotamartos","alemorcuq","agomezmoron","carrodher","dgomezleon","fmulero","FraPazGal","javsalgar","gongomgra","joancafom","marcosbc","aoterolorenzo","CeliaGMqrz","Mauraza","mdhont","migruiz4","rafariossaa"]'
```

Comment:
At this moment, not everyone in the team is providing support in the Helm charts repo, from that list, the following users are **not** providing chart support: `jotamartos`, `agomezmoron`, `gongomgra`, `aoterolorenzo`, `CeliaGMqrz`, `Mauraza`, `mdhont`

Do we want to include everyone in the support roulette?

Reply:
Removed them